crowbar_framework/app/models/backup.rb

```diff
@@ -172,6 +176,7 @@ def save_archive
     self.version = meta["version"]
     self.size = path.size
     self.created_at = Time.zone.parse(meta["created_at"])
+    self.migration_level = meta["migration_level"] || ActiveRecord::Migrator.last_migration.version
```

Comment:
>  I think it should a hard-coded version which is the latest one we have as of today.

so that we come to a state of cloud6 without updates? what would happen with that hard coded value when we add another migration?